sources/packages/web/src/store/modules/institution/actions.ts

```diff
@@ -0,0 +1,47 @@
+import { ActionTree } from "vuex";
+import { InstitutionService } from "@/services/InstitutionService";
+import {
+  InstitutionLocationState,
+  RootState,
+  InstitutionUserAndAuthDetails,
+  InstitutionLocationsDetails,
+} from "@/types";
+
+export const actions: ActionTree<InstitutionLocationState, RootState> = {
+  initialize(): boolean {
+    console.log("Initializing the institution store...");
+    return true;
+  },
+  getUserInstitutionDetails(context): Promise<InstitutionUserAndAuthDetails> {
+    return new Promise((resolve, reject) => {
```

Comment:
IMHO, we should use the async/await pattern in the same way that we have been using in others areas of the application. Is there any particular reason to use Promise callbacks here?

Reply:
![image](https://user-images.githubusercontent.com/77353155/121968802-f8297680-cd40-11eb-9cb6-72266516c535.png)
`Unnecessary try/catch wrapper` error is throwing when I use async/await with try catch